src/Servers/Kestrel/Core/src/Adapter/ConnectionAdapterContext.cs

```diff
@@ -1,11 +1,11 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.IO;
 using Microsoft.AspNetCore.Connections;
 using Microsoft.AspNetCore.Http.Features;
 
-namespace Microsoft.AspNetCore.Server.Kestrel.Core.Adapter.Internal
```

Comment:
I'm not sure we should be changing the namespace unless we're actually making the type internal.

Reply:
These honestly seemed like mistakes, I don't believe they should be pubternal to begin with (they are using in samples and public API). I can revert it for now if you prefer.

Reply:
These aren't in internal namespaces by accident. It's only a quasi-public, pubternal API currently because we didn't want to commit to a final API before.

Now is definitely not the time to commit to a final API since one of @davidfowl's goals for 3.0 is to migrate connection adapters to project bedrock connection middleware. That will hopefully mean deleting this API entirely.

Reply:
Got it, I'll do the needful.